packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -13,19 +14,42 @@
 
 <script setup lang="ts">
 import { cloneDeep } from 'lodash';
-import { computed, watch } from 'vue';
+import { computed, watch, ref } from 'vue';
 import { WorkflowNode, WorkflowPortStatus } from '@/types/workflow';
 import Button from 'primevue/button';
+import TeraProgressSpinner from '@/components/widgets/tera-progress-spinner.vue';
 import TeraOperatorPlaceholder from '@/components/operator/tera-operator-placeholder.vue';
 import { getModel, getModelConfigurationsForModel } from '@/services/model';
 import { postAsConfiguredModel } from '@/services/model-configurations';
+import { useClientEvent } from '@/composables/useClientEvent';
+import type { TaskResponse, ClientEvent } from '@/types/Types';
+import { ClientEventType, TaskStatus } from '@/types/Types';
 import { ModelConfigOperation, ModelConfigOperationState, blankModelConfig } from './model-config-operation';
 
 const props = defineProps<{
 	node: WorkflowNode<ModelConfigOperationState>;
 }>();
 const emit = defineEmits(['open-drilldown', 'append-input-port', 'update-state', 'append-output']);
 
+const documentModelConfigTaskId = ref<string>('');
+const datasetModelConfigTaskId = ref<string>('');
+
+const configModelEventHandler = async (event: ClientEvent<TaskResponse>) => {
+	const taskIdRefs = {
+		[ClientEventType.TaskGollmConfigureModel]: documentModelConfigTaskId,
+		[ClientEventType.TaskGollmConfigureFromDataset]: datasetModelConfigTaskId
+	};
+	if (event.data?.id !== taskIdRefs[event.type].value) return;
+	if ([TaskStatus.Success, TaskStatus.Cancelled, TaskStatus.Failed].includes(event.data.status)) {
+		taskIdRefs[event.type].value = '';
+	}
+};
+
+useClientEvent(ClientEventType.TaskGollmConfigureModel, configModelEventHandler);
+useClientEvent(ClientEventType.TaskGollmConfigureFromDataset, configModelEventHandler);
+
+const isLoading = computed(() => datasetModelConfigTaskId.value !== '' || datasetModelConfigTaskId.value !== '');
```

Comment:
Typo or you are checking the wrong things ... you are saying `A !== '' || A !== ''`

Reply:
yup typo